obstacle_stop_planner_refine/src/adaptive_cruise_control.cpp

```diff
@@ -120,15 +118,14 @@ AdaptiveCruiseController::insertAdaptiveCruiseVelocity(
     RCLCPP_DEBUG_THROTTLE(
       node_->get_logger(), *node_->get_clock(), std::chrono::milliseconds(1000).count(),
       "Upper velocity is too low. Insert stop line.");
-    return std::forward_as_tuple(true, output_trajectory);
+    return boost::none;
```

Comment:
I think you can use `{}` instead of `boost::none`.
If you do so, even if you change `boost::optional` to `std::optional` or `std::shared_ptr`, you don't need any change to this line.